pvm.py

```diff
@@ -36,29 +37,90 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
+
+'''
+UDP command example:
+/PVM 12:13:14
```

Comment:
now that I am seeing this here I am realizing that the current way to define and parse the timestamps are really fragile. We need to add a `timestamp` tag, like all the other commands so here it should look like
```suggestion
/PVM timestamp HH MM SS
```

Reply:
Is it possible to let MAX send a command like below
/PVM HH MM SS <action_name> <action value>
which will reduce a lot of logic complexity.

Reply:
sure thing. give me a sec.

Reply:
Thank you